sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -75,8 +75,9 @@ export interface MessageHandlers<ReceivedMessageT> {
 export { MessagingError }
 
 // @public
-export interface OperationOptions extends TracingOptions {
+export interface OperationOptions {
```

Comment:
Why do we need to remove the inheritance here? Is there something subtle at play?

Reply:
The existing `TracingOptions` is not the same as `OperationTracingOptions`. Its more like
```
interface TracingOptions {
   tracingOptions?: OperationTracingOptions
}
```

All options related to tracing will be under `tracingOptions`. We dont expect `tracingOptions` itself to have siblings also related to tracing. What is being defined upstream is the interface for `tracingOptions` which is a property in `OperationOptions`

Reply:
Makes sense! :)